images/common/bootstrap.sh

```diff
@@ -312,7 +312,7 @@ bootstrap_c8y() {
 
     DEVICE_ID=$(prompt_value "Enter the device.id" "$DEVICE_ID")
 
-    # Remove existing certificate if it does not match
+        # Remove existing certificate if it does not match
```

Comment:
Looks like some auto formatter is indenting the comments for some reason. Can you revert these changes?